### ISSUE TITLE
NullInjectorError: No provider for InjectionToken TimeLocale!

### ISSUE THREAD

**Issue by LaureneFmy** [Agranom/ngx-material-timepicker#335]
I'm trying your example on my Angular (v9) project and I'm facing this error : 

```
ERROR NullInjectorError: R3InjectorError(AppModule)[InjectionToken TimeLocale -> InjectionToken TimeLocale -> InjectionToken TimeLocale]: 
  NullInjectorError: No provider for InjectionToken TimeLocale!
```
The code :
```html
                    <div class="24hr-example">
                        <input placeholder="24hr format" aria-label="24hr format" [ngxTimepicker]="fullTime"
                            [format]="24" readonly>
                        <ngx-material-timepicker #fullTime></ngx-material-timepicker>
                    </div>
```

Am I missing something ?

**Comment by Agranom** [comment]
Have you imported module into your app.module ?

**Comment by LaureneFmy** [comment]
It's working into the app.module but why it's not on a componant's module ?

**Comment by Agranom** [comment]
Is you component's module lazy?

**Comment by LaureneFmy** [comment]
Yes it is !

**Comment by Agranom** [comment]
I have to figure it out

**Comment by emanuelevivoli** [comment]
I had the same problem.
This:
> I have to import the module into app.module and the componant's module.

Is actually what I have done.

**Comment by absalonv** [comment]
Same here. 
I had to import the module in `app.module` and in the module that is being used. 😞

**Comment by ArmaProjects** [comment]
> I have to figure it out

Is there any progress in the issue?

**Comment by Agranom** [comment]
Should be fixed in v5.5.2. Could you check pls?

**Comment by ArmaProjects** [comment]
> Should be fixed in v5.5.2. Could you check pls?
It works.
Thanks!
